src/main/java/com/thebois/models/inventory/items/ItemType.java

```diff
@@ -11,5 +11,13 @@
     /**
      * Rocks are collected from stones.
      */
-    ROCK
+    ROCK,
```

Comment:
Hmm, the resource is currently called Rock, should that change to stones or something else? Since rock -> rock is not right

Reply:
Yeah, I noticed that when I pulled in your branch a few minutes ago. 

I've renamed the Rock resource to Stone, so that it goes Stone -> Rock